### ISSUE TITLE
The loss can't be reduced when I use VOC dataset

### ISSUE THREAD

**Issue by JoyHuYY1412** [kazuto1011/deeplab-pytorch#32]
I tried to use this code for the segmentation of VOC dataset. It has about 2900 images, I use the deeplabV2 model.
The loss hardly decreases after 2k steps (I use 3 as batch size ), and it's around 0.7-1.
I don't know why.

**Comment by JoyHuYY1412** [comment]
Thank you for your reply. It works really well. I have another question, the init model, deeplabv2_resnet101_VOC2012.pth, is it trained on resnet for image classification or something?

**Comment by kazuto1011** [comment]
An init model is actually ```deeplabv2_resnet101_COCO_init.pth```, which is "ResNet-101 + ASPP" model trained on MSCOCO segmentation set. The "ResNet" part was pre-trained on ImageNet.
The ```deeplabv2_resnet101_VOC2012.pth``` that you mentioned is a model which was initialized with MSCOCO parameters above and trained on Pascal VOC 2012.
